### DISCUSSION TITLE
milestone 3-4

### DISCUSSION THREAD

**Discussion by Keiven-Soto** [uprm-inso4116-2024-2025-s1/semester-project-multiplayer-dominoes#395] [General]
Weekly meeting, scrum prep, reviewed issue

**Comment by AngelCIICMorales** [comment]
Held meeting and reviewed tasks.

**Comment by sandy-marrero** [comment]
- Completed and sent task for review: #133 
- Created M3 Slidedeck, in the process of identifying work done for this milestone to be added.
- Continuing work on the documentation tasks

**Comment by Joaquin939** [comment]
This week i worked on my final issue for this project.

**Comment by DavidCuevas3** [comment]
This week I continued planning for my final task for this milestone.

**Comment by PinkSylvie** [comment]
This week I worked on researching and reading thru the code to see how the analysis patterns have been implemented

**Comment by Saul-Caban-UPRM** [comment]
Finished 3 LTT, and finished the implementing the avatar assets just waiting for the approval of the pull request

**Comment by nsfrontera** [comment]
Working on my issues to complete them for the milestone end

**Comment by JadiellMV** [comment]
This week I worked on the profile page and profile customization, it works in localhost but not on multi-dominoes.vercel.app, working on fixing that, and also worked on my 3 LTTs.

**Comment by Adriana-Isabel** [comment]
Started to search and do the LTT.

**Comment by ebrain-mendoza121** [comment]
Kept working on lobby servers and LTT

**Comment by rukitoyordan** [comment]
Worked on LTT

**Comment by G4vp** [comment]
Worked on Lecture Topic Tasks

**Comment by Negroni1** [comment]
Worked on a refactoring code such that it follows dependency injection design pattern

**Comment by fernandopizarro1** [comment]
Finishing my LTT

**Comment by Gaby-Concepcion** [comment]
Slowed down on progress with last few issues to focus on some other exams on hopes to take this weekend and finalize everything else.

**Comment by Jvstinoo** [comment]
Working on completing my tasks.

**Comment by FrancesC8** [comment]
Continue working on assigned milestone tasks.

**Comment by AnthonyMed13** [comment]
I finished working on a task to add the roles of admin to the user model and also added a limiter to control the number of times user try to reset their passwords

**Comment by RicardoJRSs** [comment]
Worked on lecture topic tasks

**Comment by print-Eruki** [comment]
Created LTT implementation task and work on my tasks

**Comment by frances-sepulveda** [comment]
Worked on LTTs

**Comment by Adriana-Isabel** [comment]
Finished my last 3 LTTs.

**Comment by print-Eruki** [comment]
Working on finishing my task for the end
